Shared/Samples/Orbit camera around object/OrbitCameraAroundObjectView.Model.swift

```diff
@@ -0,0 +1,176 @@
+// Copyright 2024 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+extension OrbitCameraAroundObjectView {
+    /// The view model for the sample.
+    class Model: ObservableObject {
+        // MARK: Properties
+        
+        /// A scene with an imagery basemap and world elevation surface.
+        let scene: ArcGIS.Scene = {
+            let scene = Scene(basemapStyle: .arcGISImagery)
+            let elevationSource = ArcGISTiledElevationSource(url: .worldElevationService)
+            scene.baseSurface.addElevationSource(elevationSource)
+            return scene
+        }()
+        
+        /// The graphics overlay for the plane graphic.
+        let graphicsOverlay: GraphicsOverlay = {
+            let graphicsOverlay = GraphicsOverlay()
+            graphicsOverlay.sceneProperties.surfacePlacement = .relative
+            
+            let renderer = SimpleRenderer()
+            renderer.sceneProperties.headingExpression = "[HEADING]"
+            renderer.sceneProperties.pitchExpression = "[PITCH]"
+            graphicsOverlay.renderer = renderer
+            
+            return graphicsOverlay
+        }()
+        
+        /// The plane graphic created from a local URL.
+        let planeGraphic: Graphic = {
+            let planeSymbol = ModelSceneSymbol(url: .bristol, scale: 1)
+            let planePosition = Point(x: 6.637, y: 45.399, z: 100, spatialReference: .wgs84)
+            let planeGraphic = Graphic(
+                geometry: planePosition,
+                attributes: ["HEADING": 45, "PITCH": 0],
```

Comment:
Better to make it clear they are Double, not Int
```suggestion
                attributes: ["HEADING": 45.0, "PITCH": 0.0],
```